src/components/base/outline-breadcrumbs/outline-breadcrumbs.ts

```diff
@@ -0,0 +1,35 @@
+// Our base component, which all others extend.
+import { OutlineElement } from '../outline-element/outline-element';
+import { CSSResultGroup, html, css, TemplateResult } from 'lit';
+import { customElement } from 'lit/decorators.js';
+import componentStyles from './outline-breadcrumbs.css.lit';
+import { SlottedController } from '../../controllers/slotted-controller';
+
+/**
+ * The Outline  Breadcrumbs component
+ * @element Outline-Breadcrumbs
+ * @slot - The default only slot for this element.
+ */
+
+@customElement('outline-breadcrumbs')
+export class OutlineBreadcrumbs extends OutlineElement {
+  static styles: CSSResultGroup = [
+    componentStyles,
+    css`
+      span.last::after {
+        display: none;
+      }
+      span::after {
+        content: '/';
+        margin: 0 15px 10px;
```

Comment:
the Left margin here should be removed or the `mr-4` on `host::, a, ::slotted(a)` since it looks to double up the margin.

Reply:
Removed margin and things look better: 

<img width="810" alt="Screenshot of Google Chrome (9-9-21, 8-02-14 PM)" src="https://user-images.githubusercontent.com/21905/132777834-dadb2531-eae2-4019-a98d-2e17ee7620f2.png">